_layouts/news-grid.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+
+    <main id="main-content">
+       
+      <section class="usa-card__header bg-blue-80v">
+            <div class="grid-container">
+                <h1 class="font-heading-3xl text-accent-cool">{{ page.title }}</h1>
+                <h2 class="font-alt-xl text-base-lightest tablet:grid-col-9">{{ page.description }}</h2>
```

Comment:
This is functioning more as a subtitle than a heading. In that case, you can just mark up as a paragraph and uses classes to get the appearance you want. In general, we don't want to headings to style appearance alone.

Reply:
In the USWDS font size and family utility they have type-based and role-based size utilities. What are the differences between those two? For this paragraph I used the role-based class, font-alt-xl.